### PR TITLE
Update christmas_2021.py 

### PR DESCRIPTION
**Pull request by VarMonke**
This PR adds a varied embed color every time the p!event command is used[Christmas 2021]. I'd like to thank Witherr🖤#5417
on discord for helping on this PR.

### CODE REVIEW

**Comment by WitherredAway** [comment]
😳

File: cogs/christmas_2021.py [hunk 1]
```diff
@@ -69,8 +69,11 @@ async def christmas(self, ctx):
         """View Christmas event information."""
 
         member = await self.bot.mongo.fetch_member_info(ctx.author)
+        
+        def christmas_2021_ec(self):
+            return random.choice([0x9ECFFC, 0xDE2E43, 0x79B15A])
 
-        embed = self.bot.Embed(color=0xE67D23)
+        embed = self.bot.Embed(color=self.christmas_2021_ec())
```
Comment (pikaninja) [inline cogs/christmas_2021.py:76]: Is the function *really* neccesary here? Imo there's nothing wrong with simply passing the random.choice directly into color

Comment (WitherredAway) [reply cogs/christmas_2021.py:76]: we did try it but for some reason it doesn't randomize everytime the command is used, but instead everytime the cog is loaded? i dont know if i did anything wrong during the testing tho

Comment (WitherredAway) [reply cogs/christmas_2021.py:76]: i tried it again and it works, weird. thanks!

Comment (VarMonke) [reply cogs/christmas_2021.py]: Sorry for the inconvenience, it worked when we tested it out again.

**Pull request merged**